dd-java-agent/agent-profiling/profiling-auxiliary/src/main/java/com/datadog/profiling/auxiliary/AuxiliaryProfiler.java

```diff
@@ -42,9 +42,9 @@ private AuxiliaryProfiler() {
     String auxilliaryType =
         Platform.isLinux()
                 && configProvider.getBoolean(
-                    ProfilingConfig.PROFILING_ASYNC_ENABLED,
-                    isAsyncProfilerSafeInCurrentEnvironment())
-            ? "async"
+                    ProfilingConfig.PROFILING_DATADOG_PROFILER_ENABLED,
+                    isDatadogProfilerSafeInCurrentEnvironment())
```

Comment:
This should probably keep the `async-profiler` version as an alias for for backward compatibility.

Something like 
```suggestion
                    ProfilingConfig.PROFILING_DATADOG_PROFILER_ENABLED,
                    isDatadogProfilerSafeInCurrentEnvironment(),
                    ProfilingConfig.PROFILING_DEPRECATED_ASYNC_ENABLED)
```

Reply:
yes, added back